ontotext-yasgui-web-component/src/models/yasgui/yasgui.ts

```diff
@@ -13,5 +13,5 @@ export class Yasgui {
   on: (event, handler) => void;
   emitTabChange: (tab: Tab) => void;
   getOngoingRequestsInfo: (skipTabId?: string) => OngoingRequestsInfo;
-  resetResults: (resetCurrentTab: boolean) => void;
+  resetResults: (resetCurrentTab: boolean, infer?: boolean, sameAs?: boolean) => void;
```

Comment:
Create a simple model object which holds these parameters. We don't want to grow the arguments list with each new one.
Also, change the method name to `reinitYasgui` as we discussed.